string name
string value
---
bool success
string value
